cmd/jaeger/internal/extension/jaegerstorage/config.go

```diff
@@ -19,3 +21,8 @@ type MemoryStorage struct {
 	Name string `mapstructure:"name"`
 	memoryCfg.Configuration
 }
+
+func (cfg *Config) Validate() error {
+	_, err := govalidator.ValidateStruct(cfg)
```

Comment:
Memory isn't required, what's required is that at least one storage is present. You don't need to use external validator for that.

Reply:
Made changes. Please review